transaction.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The Xorm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xorm
+
+// AutoTransaction Execute sql wrapped in a transaction(abbr as tx), tx will automatic commit if no errors occurred
+func AutoTransaction(f func(*Session) (interface{}, error), engine *Engine) (interface{}, error) {
```

Comment:
I think it's better to let `AutoTransaction` become a method of `Engine` and the name `AutoTransaction` -> `Transaction`?

Reply:
got it